src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -455,21 +538,33 @@ struct LottiePathSet : LottieProperty
         return (*frames)[frames->count];
     }
 
-    bool operator()(float frameNo, Array<PathCommand>& cmds, Array<Point>& pts, Matrix* transform)
+    bool operator()(float frameNo, Array<PathCommand>& cmds, Array<Point>& pts, Matrix* transform, float roundness)
     {
         if (!frames) {
+            if (roundness > ROUNDNESS_EPSILON) {
+                _handleCorners(value, cmds, pts, transform, roundness);
```

Comment:
These handleCorners could be inlined, could you please confirm the binary size? probably, implementing functions in cpp file might be better for it. Here is a way to check the bin size.
```
$strip libthorvg.so.xxx
$size libthorvg.so.xxx
```

Reply:
after introducing cpp the bin size is bigger (if I inline the fun and if I don't)